src/main/scala/ai/eto/rikai/sql/spark/parser/RikaiExtAstBuilder.scala

```diff
@@ -66,6 +66,31 @@ private[parser] class RikaiExtAstBuilder
     )
   }
 
+  override def visitOptionList(ctx: OptionListContext): Map[String, String] =
+    ctx match {
+      case null => Map.empty
+      case _    => ctx.option().asScala.map(visitOption).toMap
+    }
+
+  override def visitOption(ctx: OptionContext): (String, String) = {
```

Comment:
Need some scala help here @changhiskhan

Reply:
I also find this, maybe we can try in parser https://stackoverflow.com/questions/33281312/how-to-strip-quotes-from-a-string